src/models/employee.ts

```diff
@@ -0,0 +1,5 @@
+export interface Employee {
```

Comment:
```suggestion
export interface IEmployee {
```

As per comment on Customer interface.

Reply:
Applied changes